tools/envoy/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM envoyproxy/envoy-alpine:v1.14.3
+FROM marcinfalkowski/envoy-dev:v1.16.1-dev-lua-segfault-fix-1-16-0-backport-20201118-df9dc819
 
 ENV PORT=9999:9999
 ENV PORT=80:80
 ENV ENVOY_NODE_ID=front-proxy-id
 ENV ENVOY_NODE_CLUSTER=front-proxy
-ENV ENVOY_EGRESS_LISTENER_PORT=80
-ENV ENVOY_INGRESS_LISTENER_PORT=8080
+ENV ENVOY_EGRESS_LISTENER_PORT=31000
```

Comment:
Is the port change relevant / needed?

Reply:
yes, new envoy container versions launch envoy from user `envoy`, not `root`, which has no access to low ports